test/essential/ad.jl

```diff
@@ -95,15 +95,14 @@
         sample(dir(), HMC(0.01, 1), 1000)
         Turing.setrdcache(false)
     end
-    # FIXME: For some reasons PDMatDistribution AD tests fail with ReverseDiff
     @testset "PDMatDistribution AD" begin
         @model function wishart()
             theta ~ Wishart(4, Matrix{Float64}(I, 4, 4))
         end
         Turing.setadbackend(:tracker)
         sample(wishart(), HMC(0.01, 1), 1000);
-        #Turing.setadbackend(:reversediff)
-        #sample(wishart(), HMC(0.01, 1), 1000);
+        Turing.setadbackend(:reversediff)
```

Comment:
I also came across these not too long ago due to the changes we've been making in Bijectors. I think it might just be due to https://github.com/TuringLang/Bijectors.jl/pull/246